### PR TITLE
Move alert banner to the top

### PR DESCRIPTION
**Pull request by ajinkyaraj-23**

### CODE REVIEW

**Review by rickykuo666 — DISMISSED** [review]
LGTM

File: components/icons/WarningIcon.tsx [hunk 1]
```diff
@@ -1,7 +1,12 @@
 import React from 'react'
 import { Icon, IconProps } from '@chakra-ui/react'
 
-export const WarningIcon = ({ ...styles }: IconProps) => {
+export const WarningIcon = ({
+  color,
```
[the remark this answers is not in the document]
Comment (rickykuo666) [reply components/icons/WarningIcon.tsx:5]: and add this `   {...styles}` to Icon

File: components/DisabledStakeAlert.tsx
[the remark this answers is not in the document]
Comment (rickykuo666) [reply components/DisabledStakeAlert.tsx:1]: so can remove `StakingAlertBox` and its just like this 

```
export const DisabledStakeAlert = ({
  opStatus,
  acctInfo,
  ...styles
}: {
  opStatus: StakingOpsStatus
  acctInfo: AccountInfo | null
} & BoxProps) => {
  let backgroundColor = 'gray.200'

  if (opStatus.Delegated && !opStatus.CanStake) {
    if (!opStatus.bakerAcceptsStaking && (acctInfo?.stakedBalance ?? 0) > 0) {
      backgroundColor = 'red.200'
    }
    return (
      <Box w='100%' borderLeft='4px solid gray' p={4} {...styles}>
        <Flex alignItems='center' gap='12px'>
          <WarningIcon fill={backgroundColor} />
          <Text fontSize='16px' lineHeight='22px' color='#2D3748'>
            {getDisabledStakeButtonReason(opStatus)}
          </Text>
        </Flex>
      </Box>
    )
  }
}

```

**Review by rickykuo666 — CHANGES_REQUESTED** [review]

File: components/icons/WarningIcon.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply components/icons/WarningIcon.tsx]: Done

File: components/DisabledStakeAlert.tsx
[the remark this answers is not in the document]
Comment (ajinkyaraj-23) [reply components/DisabledStakeAlert.tsx:1]: Done.

**Pull request merged**